docs/ValidationCriteriaMaster.md

```diff
@@ -0,0 +1,426 @@
+| IfcCourse       | Segmento di massicciata | PSet_CourseCommon            | NominalThickness             | IfcNonNegativeLengthMeasure |                                                                                                    | IfcPropertySingleValue     |
+| IfcCourse       | Segmento di massicciata | PSet_CourseCommon            | NominalWidth                 | IfcNonNegativeLengthMeasure |                                                                                                    | IfcPropertySingleValue     |
+| IfcCourseType   | Segmento di massicciata | RFI_S22600                   | Tipo massicciata             | IfcLabel                    | Misto, Calcareo, Tenace                                                                            | IfcPropertyEnumeratedValue |
+| IfcGroup        | Segmento di rotaia      | RFI_S22700                   | Galleria                     | IfcLogical                  |                                                                                                    | IfcPropertySingleValue     |
+| IfcGroup        | Segmento di rotaia      | RFI_S22700                   | Tipo binario di appartenenza | IfcLabel                    | BINARIO CENTRALIZZATO, BINARIO DI CORSA, BINARIO DI FASCIO, BINARIO IMP. SMISTAMENTO AUTOMATICO    | IfcPropertyEnumeratedValue |
+
+</details>
+
+<details><summary>Groups Spatial Reference Table template</summary>
+
+| **Spatial Element** | **Spatial Element Type** | **Minimum** | **Maximum** | **Product or Group** | **Product Type or Group Type** |
+|---------------------|--------------------------|-------------|-------------|----------------------|--------------------------------|
+| IfcRailway          | Località                 | 1           | 1           | IfcGroup             | Binari di corsa (Contenitore)  |
+| IfcFacilityPart     | TRACKSTRUCTURE           | 1           | 1           | IfcGroup             | Deviatoi                       |
+| IfcFacilityPart     | TRACKSTRUCTURE           | 1           | 1           | IfcGroup             | Massicciata                    |
+| IfcFacilityPart     | TRACKSTRUCTURE           | 1           | 1           | IfcGroup             | Rotaie                         |
+| IfcFacilityPart     | TRACKSTRUCTURE           | 1           | 1           | IfcGroup             | Traverse                       |
+
+</details>
+
+<details><summary>Spatial Decomposition Table template</summary>
+
+> OPTION 1: Generic, just with types
+
+| **Parent Element** | **Parent Element Type** | **Minimum** | **Maximum** | **Child Element** | **Child Element Type** |
+|--------------------|-------------------------|-------------|-------------|-------------------|------------------------|
+| IfcRailway         | Località                | 2           | 2           | IfcFacilityPart   | TRACKSTRUCTURE         |
+
+> OPTION 2: Specific, with occurrences' names
+
```

Comment:
I'd merge the two options and add a column **Child Element Names** and just list them.

Reply:
done